std/socket.d

```diff
@@ -3522,8 +3522,8 @@ class UdpSocket: Socket
             @property @trusted Address localAddress() { assert(0); }
             @trusted ptrdiff_t send(const(void)[] buf, SocketFlags flags) { assert(0); }
             @safe ptrdiff_t send(const(void)[] buf) { assert(0); }
-            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, Address to) { assert(0); }
-            @safe ptrdiff_t sendTo(const(void)[] buf, Address to) { assert(0); }
+            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, in Address to) { assert(0); }
+            @safe ptrdiff_t sendTo(const(void)[] buf, in Address to) { assert(0); }
```

Comment:
Your change breaks classes which inherit from Socket and override its methods (`sendTo`, here). This test ensures that this doesn't happen, so it should not be changed without a strong justification for breaking existing code. See [https://issues.dlang.org/show_bug.cgi?id=16514](issue 16514) for details.